packages/grafana-data/src/types/suggestions.ts

```diff
@@ -0,0 +1,8 @@
+import { DataFrame, PanelPlugin, VariableModel, VariableSuggestion, VariableSuggestionsScope } from '..';
+
+export type SuggestionsProvider = (
```

Comment:
Introducing this type for having unified providers for panel/field option suggestions